specs/phase0/p2p-interface.md

```diff
@@ -846,13 +844,11 @@ The response MUST consist of a single `response_chunk`.
 
 ## The discovery domain: discv5
 
-Discovery Version 5 ([discv5](https://github.com/ethereum/devp2p/blob/master/discv5/discv5.md)) is used for peer discovery.
+Discovery Version 5 ([discv5](https://github.com/ethereum/devp2p/blob/master/discv5/discv5.md)) (Protocol version v5.1) is used for peer discovery.
```

Comment:
The contents of this link still say "draft October 2019", even thought the other documents are updated to v5.1. Maybe needs an update? Link has no version and won't break, so ok for this PR.

Reply:
https://github.com/ethereum/devp2p/pull/165